### PR TITLE
https://issues.redhat.com/browse/ACM-12000

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: install/uninstall.adoc [hunk 1]
```diff
@@ -26,33 +28,58 @@ To disable Discovery, complete the following steps:
 - You can also use the terminal. Run the following command to disable Discover:
 
 +
+[source,bash]
+----
+oc delete discoveryconfigs --all --all- namespaces
```
Comment (dockerymick) [inline install/uninstall.adoc]: @dislbenn is this command correct?

Comment (dislbenn) [reply install/uninstall.adoc]: @dockerymick that command looks correct; however, there should be no space in `--all-namespaces`.

Comment (dockerymick) [inline install/uninstall.adoc]:
```suggestion
oc delete discoveryconfigs --all --all-namespaces
```

**Comment by dockerymick** [comment]
Previously merged PRs: https://github.com/stolostron/rhacm-docs/pull/6562, https://github.com/stolostron/rhacm-docs/pull/6563, and https://github.com/stolostron/rhacm-docs/pull/6515/files

Merging this PR

**Pull request merged**